### DISCUSSION TITLE
Stolpersteine

### DISCUSSION THREAD

**Discussion by OpenGreenStreet** [streetcomplete/StreetComplete#4051] [Ideas]
Warum werden https://de.wikipedia.org/wiki/Stolpersteine nicht erfasst/abgefragt?

**Comment by FloEdelmann** [comment]
Siehe #2115 (wurde noch nicht gemerged), insbesondere https://github.com/streetcomplete/StreetComplete/pull/2115#issuecomment-700622438.

**Reply by OpenGreenStreet** [reply]
@FloEdelmann , danke, gibt es einen Zeithorizont?

**Reply by FloEdelmann** [reply]
Nein, siehe https://github.com/streetcomplete/StreetComplete/pull/2115#issuecomment-1028757516.

Aber die fehlenden Fotos zu suchen (oder auch die Arbeit aus #2115 und #1572 in der Issue zusammenzufassen) kann helfen, schneller einen neuen PR möglich zu machen.

**Reply by OpenGreenStreet** [reply]
noch eine Idee, wenn sie nicht geäußert wurde: wie beim Vermessen der Straßen per Kamera die Stolpersteine fotografieren und anschließend über den Server per Texterkennung die Daten der Stolpersteine auslesen und übernehmen.

**Reply by FloEdelmann** [reply]
Let me answer in English, since this may be relevant for others reading this, too:

I haven't heard this suggestion yet, but it'll probably not get accepted:

1. Usefulness: The text on each Stolperstein is probably not too useful for data consumers. If there are use cases, please comment them.
2. Offline quests: Having to upload images to a server for OCR (optical character recognition) means the quest can't be solved while offline. It might be done when uploading the quest answers (like it's done when leaving a note with photo), but then the OCR result can't be verified by the user.
3. Complexity: This is a totally new type of quest, so it'll be quite a lot of work to implement.

Another idea would be to ask for the photo directly instead of the text. There are some similar quest suggestions which were not accepted: #2762, #1601, #1883, #1515

**Reply by OpenGreenStreet** [reply]
@FloEdelmann, Thank you for the detailed feedback. To ask for a photo, I would find very useful. From my point of view, the location of the "Stolperstein" the name of the person and a photo should be recorded.

**Reply by FloEdelmann** [reply]
> the location

Note that StreetComplete cannot add new elements to OSM, as quests always have to be asked for existing elements.

> To ask for a photo, I would find very useful.

Please check https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md
If it passes, feel free to open a new issue and fill out the quest suggestion template, so it can be discussed in more detail:
https://github.com/streetcomplete/StreetComplete/issues/new?assignees=&labels=&template=quest-suggestion.md

**Comment by The-Nickname** [comment]
Sollte nicht wenigstens bei bereits Existierenden Stolpersteinen ohne Namen nach dem Namen gefragt werden?

**Reply by rusty-snake** [reply]
Klingt im ersten Moment nicht verkehrt, wäre aber ein neuer Quest -> https://github.com/streetcomplete/StreetComplete/issues/new?template=quest-suggestion.md

edit: und https://github.com/streetcomplete/StreetComplete/issues/6042

**Reply by OpenGreenStreet** [reply]
Heißt das, ich sollte eine entsprechenden neuen Quest anlegen?

**Comment by SkaveRat** [comment]
Neuerdings werden Stolpersteine mit dem "Was für ein Kunstwerk ist das" Quest abgefragt.

Was wäre hier die korrekte Antwort?

**Reply by rusty-snake** [reply]
Ref. #6777
